changelog/11942.txt

```diff
@@ -1,3 +1,3 @@
-```release-note:improvement
-plugins/ad: Added rotate-role endpoint for manual service account password rotations
+```release-note:feature
+secrets/ad: Add rotate-role endpoint for manual service account password rotations
```

Comment:
I am unsure, but this doesn't look quite right. Unless I am reading the changelog formatting wiki too literally. (which I might be doing)

But I think this should be


```suggestion
**Active Directory service account manual rotation**: Add rotate-role endpoint for manual service account password rotations
```

Reply:
Yep that's right! If we're changing it to be a feature, it should be in the format that @fairclothjm is suggesting.